### PR TITLE
SISRP-48237: Notifications Card, stage 2

### PR DESCRIPTION
**Pull request by developish**
https://jira-secure.berkeley.edu/browse/SISRP-48237

### CODE REVIEW

File: src/react/components/_dashboard/NotificationsCard/Tabs.module.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import 'src/react/stylesheets/variables';
+@import 'src/react/stylesheets/mixins';
+
+$height: 42px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/Tabs.module.scss]: Just asking simple question, @developish -- should this be put into `variables` and not hard-coded?

But maybe you have reason why not?

File: src/react/components/_dashboard/NotificationsCard/DateLabel.module.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import 'src/react/stylesheets/variables';
+
+.dateLabel {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  width: 40px;
+  margin-right: 15px;
+  padding-top: 15px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/DateLabel.module.scss]: You can use `$large-spacing`

File: src/react/components/_dashboard/NotificationsCard/BCourses/SourceAndTypeGroup.module.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+.sourceLabel {
+  font-size: 11px;
+  line-height: 13px;
+  color: #666;
+}
+
+.secondaryLabel {
+  font-size: 13px;
+  line-height: 15px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/BCourses/SourceAndTypeGroup.module.scss]: You can use `$large-spacing`

File: src/react/components/_dashboard/NotificationsCard/BCourses/SourceAndTypeGroup.module.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  color: #333;
+  padding-top: 5px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/BCourses/SourceAndTypeGroup.module.scss]: You can use `$small-spacing`

File: src/react/components/_dashboard/NotificationsCard/BCourses/MultiNotifications.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+import Linkify from 'react-linkify';
+import styles from './SourceAndTypeGroup.module.scss';
+
+import GradedLabel from './GradedLabel';
+import CountBadge from './CountBadge';
+
+import Notification from './notification';
+
+const MiniNotification = ({ notification, expanded, setExpanded }) => {
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/BCourses/MultiNotifications.js]: Does this need a `.displayName`?

File: src/react/components/_dashboard/NotificationsCard/BCourses/BCoursesTab.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+import { connect } from 'react-redux';
+import { parseISO } from 'date-fns';
+
+import { groupByDate } from '../notifications.module';
+
+import DisclosureChevron from 'react/components/DisclosureChevron';
+import NoNotfications from '../NoNotifications';
+import DateGroup from './DateGroup';
+
+const BCoursesNotifications = ({ notifications, shownCount, showMore }) => {
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/BCourses/BCoursesTab.js]: `.displayName`?

File: src/icons/review.svg [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (marcpechaitis) [inline src/icons/review.svg]: Weren't we talking about trying to clean up/optimize (ie, svgo) our SVGs?

Comment (developish) [reply src/icons/review.svg]: Are you talking about doing that as part of the build? This is just one I received from the designer (that is, not exported from Figma - so, better?)

File: src/react/components/_dashboard/NotificationsCard/Tabs.module.scss (hunk 1 above)
Comment (developish) [reply src/react/components/_dashboard/NotificationsCard/Tabs.module.scss]: I consider this to be a _local_ variable to this file. 42px high doesn't mean anything particular to put it in the global variables file

File: src/icons/review.svg (hunk 7 above)
Comment (marcpechaitis) [reply src/icons/review.svg]: I guess I was expecting the designers to provide optimized SVGs?

File: src/react/components/_dashboard/NotificationsCard/Tabs.module.scss [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+.tab {
+  font-weight: bold;
+  flex: 1;
+  text-align: center;
+  line-height: $height;
+}
+
+.currentTab {
+  color: $activeColor;
+  border-bottom: 2px solid $activeColor;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/Tabs.module.scss]: `$tiny-spacing`

File: src/react/components/_dashboard/NotificationsCard/Tabs.module.scss [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.inactiveTab {
+  color: $inactiveColor;
+  border-bottom: 2px solid $inactiveBorderColor;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/Tabs.module.scss]: `$tiny-spacing`

File: src/react/components/_dashboard/NotificationsCard/University/Messages.module.scss [hunk 10]
```diff
@@ -0,0 +1,24 @@
+@import 'src/react/stylesheets/variables';
+
+.messages {
+  flex: 1;
+}
+
+.message {
+  padding-top: 15px;
+  padding-bottom: 15px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/University/Messages.module.scss]: `$large-spacing`

File: src/react/components/_dashboard/NotificationsCard/University/Unread.module.scss [hunk 11]
```diff
@@ -0,0 +1,9 @@
+$height: 22px;
+
+.banner {
+  margin-top: 15px;
```
Comment (marcpechaitis) [inline src/react/components/_dashboard/NotificationsCard/University/Unread.module.scss]: `$large-spacing`

**Comment by marcpechaitis** [comment]
Nice work @developish, I think the nitpicking in my review shouldn't hold this up from going forward.

**Pull request merged**